universal-application-tool-0.0.1/app/forms/DropdownQuestionForm.java

```diff
@@ -0,0 +1,15 @@
+package forms;
+
+import services.question.DropdownQuestionDefinition;
+import services.question.QuestionType;
+
+public class DropdownQuestionForm extends MultiOptionQuestionForm {
+
+  public DropdownQuestionForm() {
+    super(QuestionType.DROPDOWN);
```

Comment:
Ohh, I love this update.

Reply:
Ah, I realize this wasn't an "update" - but I think we should use this style in the `QuestionForm` too! To have the constructor take the type. Way cleaner.

Reply:
To be clear, that's not a blocker for this PR, just thinking aloud...

Reply:
Good idea! The Play `Form`s need parameterless-constructors, so this was my solution